apps/earn-protocol-landing-page/components/layout/LandingMasterPage/landingPageBlobs.module.css

```
@@ -0,0 +1,356 @@
+.blobsContainer {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  overflow: hidden;
+  pointer-events: none;
+  z-index: 0;
+}
+
+@keyframes driftSmall1 {
```

Stylelint (17.5.0)

[error] Expected keyframe name "driftSmall1" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)